crates/humanode-peer/src/chain_spec.rs

```diff
@@ -344,7 +356,15 @@ fn testnet_genesis(
             accounts: {
                 let evm_pot_accounts = vec![(
                     humanode_runtime::EvmToNativeSwapBridgePot::account_id(),
-                    evm_genesis_account(INITIAL_POT_ACCOUNT_BALANCE),
+                    evm_genesis_account(
+                        DEV_ACCOUNT_BALANCE
+                            .checked_mul(endowed_accounts.len().try_into().unwrap()).unwrap()
+                            .checked_add(INITIAL_POT_ACCOUNT_BALANCE.checked_mul(pot_accounts.len().try_into().unwrap()).unwrap()
+                            .checked_add(
+                                // Own bridge pot minimum balance.
+                                <humanode_runtime::EvmBalances as frame_support::traits::Currency<EvmAccountId>>::minimum_balance(),
+                            ).unwrap()).unwrap()
+                    ),
```

Comment:
Can we place this calculation in a separate fn and make it more explicit and testable? The same goes for the formula for the native account bridge

Reply:
Got it, sure. But what do you mean in term of `testable`? As it's tested in `chain_spec` tests.

Reply:
The fn itself could have a number of unit tests focusing on illustrating why that fn works right.

Reply:
Redesigned in a little bit different way. Please, check.

Reply:
Well, this does not help with the growing complexity at `testnet_genesis` fn.

Reply:
https://github.com/humanode-network/humanode/pull/700/commits/6daee15de13c754f806225d468c20cf60848edc5